Tasks/Ann_Valenda/task7/data.py

```diff
@@ -0,0 +1,112 @@
+import json
```

Comment:
the storage is very good for the learning purposes, but keep in mind that it will not be often used in production